src/Components/DrillPage.js

```diff
@@ -32,25 +42,30 @@ export class DrillPage extends Component {
           </TouchableOpacity>
         </ImageBackground>
         <View style={styles.separator} />
-
-        <Button title="Favoris" onPress={() => this._toggleFavorite()} />
+        <TouchableOpacity
+          style={styles.favoriteContainer}
+          onPress={() => this.props.toggleFavorite(drill)}
```

Comment:
You can probably directly call `this.props.toggleFavorite` without the arrow function

Reply:
Done for var
The second one doesn't work